org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/grammarAccess/GrammarAccessExtensions.xtend

```diff
@@ -66,9 +65,6 @@ class GrammarAccessExtensions {
 		"\t" -> "tab",
 		"\\" -> "backslash"
 	};
-	val Map<String, ISerializer> xtextSerializerByLineDelimiter = Maps.newHashMapWithExpectedSize(2)
-	
-	@Inject CodeConfig codeConfig
```

Comment:
I thought we'd need that for the line ending?

Reply:
ok